native/cocos/bindings/manual/jsb_cocos_manual.cpp

```diff
@@ -36,7 +36,7 @@
 
 extern se::Object *__jsb_cc_FileUtils_proto; // NOLINT(readability-redundant-declaration, readability-identifier-naming)
 
-static bool jsb_ccx_empty_func(const se::State & /*s*/) { // NOLINT(readability-identifier-naming)
+static bool jsb_ccx_empty_func(se::State & /*s*/) { // NOLINT(readability-identifier-naming)
```

Comment:
Why remove const?

Reply:
All jsb function should be unified and meet the signature `using se_function_ptr = bool (*)(se::State &state);` as an parameter.

Reply:
ok